authentication.md

```diff
@@ -16,21 +16,21 @@ The framework integration will provide the application with some means of resolv
 
 * Reading the request headers for authentication information
 * Resolving the access token or API key presented to a Stormpath account
-* Return this information to the application, along with the method of authentication.
+* Return this information to the application, along with information about how the request was authenticated.
 
 This information will allow the application to make authorization decisions for the request. 
 
 **Unauthenticated Requests**
 
-In addition, the framework integration MAY provide the application with some means of rendering errors for unauthenticated requests. This may be combined with the Account Resolvers for extra convenience. 
+As a convenience, the framework integration MAY provide the application with a means to, in one step, check for authentication information and render an error if the request is unauthenticated. 
```

Comment:
> Just curious: what else are you thinking of for the "unauthenticated request logic" besides rendering the view?

Well, it's not always strictly rendering. It also wraps up the content negotiation logic and decides whether to return a JSON or HTML response. Plus, it figures out what URL to redirect to (if the Stormpath configuration default of `/login` has changed), and includes whatever method of URL-forwarding is available.